src/vs/workbench/api/common/extHostTesting.ts

```diff
@@ -138,10 +138,12 @@ export class ExtHostTesting implements ExtHostTestingShape {
 			createTestRun: (request, name, persist = true) => {
 				return this.runTracker.createTestRun(controllerId, collection, request, name, persist);
 			},
-			invalidateTestResults: item => {
+			invalidateTestResults: items => {
 				checkProposedApiEnabled(extension, 'testInvalidateResults');
-				const id = item ? TestId.fromExtHostTestItem(item, controllerId).toString() : controllerId;
-				return this.proxy.$markTestRetired(id);
+				for (const item of items instanceof Array ? items : [items]) {
```

Comment:
I have been doing `Array.isArray(arr)` for ages. Is `items instanceof Array` better?

Reply:
Iirc `Array.isArray(arr)` works if the array is coming from a different origin. I generally like `instanceof Array` since it's simpler and more consistent 🤷 

Probably the nice way to do this now would be checking if the object has `[Symbol.iterator]`, but we don't yet take Iterables into any methods in extension host APIs.